README.md

```diff
@@ -1,36 +1,19 @@
 # Krun
 
-Krun is a framework for running software benchmarking experiments.
+Krun is a framework for running high-quality software benchmarking experiments.
+Krun experiments consist of a configuration file, a carefully configured
+benchmarking machine, and the Krun program itself.
 
-**Krun uses sudo to elevate privileges! Please read these instructions in
-full.**
 
-The `examples/` directory contains a simple experiment using Krun.
-This is a good starting point for setting up your own Krun configuration.
+## Step 1: Initial installation
```

Comment:
Title case all titles.

Reply:
They weren't title-cased before...

Reply:
Oops. Well up to you if you change them.